doc/changelog.md

```diff
@@ -1,30 +1,208 @@
-# Thrust v1.9.8 (CUDA 11.0)
+# Thrust 1.9.10 (NVIDIA HPC SDK 20.5)
 
 ## Summary
 
-Thrust v1.9.8, which is included in the CUDA 11.0 release, removes Thrust's
-  internal derivative of CUB, upstreams all relevant changes too CUB, and adds
-  CUB as a Git submodule.
+Thrust 1.9.10 is the release accompanying the NVIDIA HPC SDK 20.5.
+It adds CMake support for compilation with NVC++ and a number of minor bug fixes
+  for NVC++.
+It also adds CMake `find_package` support.
+
+## New Features
+
+- #1130: CMake `find_package` support.
+  This is significant because there is a legacy `FindThrust.cmake` script
+    authored by a third party in widespread use in the community which has a
+    bug in how it parses Thrust version numbers which will cause it to
+    incorrectly parse 1.9.10.
+  This script only parses the first digit of each part of the Thrust version
+    number: for example, Thrust 17.17.17 would be interpreted as Thrust 1.1.1.
```

Comment:
This is not accurate; it also assumed fixed number of `0`s between the significant digits. We should probably just say that it assumed single digit version components and broke down when that was not the case.

Reply:
I'd still like to have an example. Can we just think up an accurate one?

Reply:
I think 17.17.17 would make it actually not match and create a malformed version string. Should be easy to test.

Reply:
I changed the version int to `101009` locally, got this:
```
-- Found Thrust: /usr/local/cuda/include (found version "1.1.09") 
```

Changed it to `1701717`, which I believe is the correct version literal for 17.17.17, got this:
```
-- Found Thrust: /usr/local/cuda/include (found version "1.1.1701717") 
```

For `1000100` (so 10.1.0):
```
-- Found Thrust: /usr/local/cuda/include (found version "1.0.0100") 
```